web-marketplace/src/pages/EcocreditsByAccount/CreditBatchesTab/CreditBatchesTab.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box } from '@mui/system';
+import { Link } from 'react-router-dom';
+
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import EmptyState from 'web-components/lib/components/empty-state';
+import NoEcocreditsIcon from 'web-components/lib/components/icons/NoEcocreditsIcon';
+import PlusIcon from 'web-components/lib/components/icons/PlusIcon';
+import { Label, Subtitle } from 'web-components/lib/components/typography';
+import { CreditBatches } from 'components/organisms';
+
+import WithLoader from 'components/atoms/WithLoader';
+import { useProfileData } from '../hooks/useProfileData';
+import { usePaginatedBatchesByIssuer } from 'hooks/batches/usePaginatedBatchesByIssuer';
+import { useTheme } from '@mui/material';
+import { PROFILE_NO_CREDIT_BATCHES_MESSAGE } from './CreditBatchesTab.constants';
+
+export const CreditBatchesTab = () => {
+  const { address, party, isLoading: isLoadingProfileData } = useProfileData();
+  const { batchesWithSupply, setPaginationParams } =
+    usePaginatedBatchesByIssuer({ address });
+  const theme = useTheme();
+
+  const hasNoBatches = batchesWithSupply && batchesWithSupply?.length === 0;
+
+  return (
+    <Box sx={{ width: '100%' }}>
```

Comment:
Actually, it's not the same thing, the design is different, and `MyCreditbatches` has an additional create button. If it's fine I can merge the design and toggle the button with a prop.

Reply:
Ok I did it here:  [a96f7ca](https://github.com/regen-network/regen-web/pull/2141/commits/a96f7cad76d4594db3da16784fde7da146a1ea3b)
I think it will be more consistent this way.

Reply:
> Actually, it's not the same thing, the design is different, and `MyCreditbatches` has an additional create button. If it's fine I can merge the design and toggle the button with a prop.

The private state for the credit batches table was updated in the comps since this was implemented, so ideally the private and public state would look the same in terms of the header (private state: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?type=design&node-id=5255%3A166692&mode=design&t=sru9OFFCOrXo7e3Z-1)

Reply:
Ok great, you can check that it has been updated here: https://deploy-preview-2141--regen-marketplace.netlify.app/profile/credit-batches

Reply:
Ok looks good, just the tabs are off: https://monosnap.com/file/WsLkLJj9cbo8V9QXt5k7O1LQv6uVPX

Reply:
This is fixed.